index.js

```diff
@@ -0,0 +1,23 @@
+const hamburgerEl = document.querySelector(".hamburger-menu");
+const mobilemenuEl = document.querySelector(".menu__mobile-none");
+const bodyEl = document.querySelector("body");
+
+//functions
+const hamburgerClickHandler = () => {
+    console.log("hi")
+    
+    hamburgerEl.classList.contains("click") ? 
+        hamburgerEl.classList.remove("click") : hamburgerEl.classList.add("click")
```

Comment:
there is a `classList.toggle`. No need to update this if you don't want to. Just check it out when you get time.